packages/lodestar/src/eth1/wallet.ts

```diff
@@ -24,20 +24,28 @@ export class Eth1Wallet {
 
   private logger: ILogger;
 
+  private withdrawalKey: PrivateKey;
+
+  private signingKey: PrivateKey;
+
   public constructor(
-    privateKey: string,
+    eth1PrivateKey: string,
     contractAbi: string|ParamType[],
     config: IBeaconConfig,
     logger: ILogger,
-    provider?: Provider
+    provider?: Provider,
+    withdrawalKey?: PrivateKey,
+    signingKey?: PrivateKey,
   ) {
     this.config = config;
     this.logger = logger;
     if(!provider) {
       provider = ethers.getDefaultProvider();
     }
-    this.wallet = new Wallet(privateKey, provider);
+    this.wallet = new Wallet(eth1PrivateKey, provider);
     this.contractAbi = contractAbi;
+    this.withdrawalKey = withdrawalKey || PrivateKey.random();
+    this.signingKey = signingKey || PrivateKey.random();
```

Comment:
This is eth1 wallet, it makes more sense to receive optional eth2 keys in `createValidatorDeposit`

Reply:
The previous logic had this as BLS keys?

Reply:
Not sure

Reply:
Have a look lower down at the old code, the withdrawal and signing key were the exact same. So I separated it

Reply:
Oh, yeah, but still it doesn't make much sense for eth1 wallet to hold eth2 keys in my opionion

Reply:
Yeah, instead of passing the keys in the constructor, pass them in the `createValidatorDeposit` call.